client/components/ScrollableContentWrapper.tsx

```diff
@@ -1,24 +1,24 @@
-import React, { CSSProperties, FC } from 'react';
+import React, { CSSProperties, FC, useMemo } from 'react';
 import SimpleBar from 'simplebar-react';
 import 'simplebar/src/simplebar.css';
 
 import { useDir } from '../hooks/useDir';
 
 
 const styleDefault = {
-	maxHeight: '100%', flexGrow: 1,
+	maxHeight: '100%', flexGrow: 1, overflowY: 'auto', willChange: 'transform',
 };
 
-
 type CustomScrollbarsProps = {
 	onScroll?: Function;
 	ref: React.Ref<unknown>;
 	style?: CSSProperties;
 }
 
-const ScrollableContentWrapper: FC<CustomScrollbarsProps> = React.memo(React.forwardRef(({ onScroll, children, style = styleDefault }, ref) => {
+const ScrollableContentWrapper: FC<CustomScrollbarsProps> = React.memo(React.forwardRef(({ onScroll, children, style }, ref) => {
```

Comment:
I'm a little against use `React.*()` calls. Not only the next releases of React will not come with a default value, but right now we prefer `useMemo()` over `React.useMemo()`. Therefore, I think `memo()` and `forwardRef()` should be adopted. What do you think?